cmd/kms-handlers.go

```diff
@@ -223,6 +230,22 @@ func (a kmsAPIHandlers) KMSDeleteKeyHandler(w http.ResponseWriter, r *http.Reque
 // KMSListKeysHandler - GET /minio/kms/v1/key/list?pattern=<pattern>
 func (a kmsAPIHandlers) KMSListKeysHandler(w http.ResponseWriter, r *http.Request) {
 	ctx := newContext(r, w, "KMSListKeys")
+	if GlobalKMS != nil {
+		if GlobalKMS.IsLocal() {
+			secretKey := GlobalKMS.ListLocal()
```

Comment:
We could simply call this `List()` not sure why we need ListLocal() a List simply returns what is the default keyID.